docs/pages/application-access/introduction.mdx

```diff
@@ -1,13 +1,12 @@
 ---
-title: Application Access Guide
-description: How to set up and configure Teleport for Application access with SSO and RBAC
+title: Protect Applications with Teleport
+description: How to set up Teleport to protect internal apps and cloud provider APIs
 ---
 
-# Application Access
-
 Teleport Application Access is designed to provide secure access to
-internal dashboards and applications, such as:
+cloud provider APIs and internal applications. Examples include: 
 
+- The AWS management console.
```

Comment:
Should we add another bullet point for Azure and GCP cloud APIs?

Reply:
I've added a bullet for CLIt tools for all three major cloud providers